### PR TITLE
Fix CLI processes emitting empty root spans when CLI tracing is not enabled

### LINKED ISSUE #1314: [BUG]  Fixing malformed trace after upgrade from 0.54.0 to 0.63.0
### Bug description
After upgrade **datadog-php-tracer** from  **0.54.0** to **0.63.0** version we are experiencing some problems.
A lot of `Fixing malformed trace`  messages in the log file. And sometimes a segfault appears

### PHP version
```
PHP 8.0.10 (cli) (built: Aug 26 2021 15:50:07) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.10, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.10, Copyright (c), by Zend Technologies
    with ddtrace v0.63.0, Copyright Datadog, by Datadog
```

### Installed extensions
<details>
<summary>PHP extensions</summary>

```
[PHP Modules]
amqp
bcmath
calendar
Core
ctype
curl
date
ddtrace
dom
exif
FFI
fileinfo
filter
ftp
gd
gettext
grpc
hash
iconv
igbinary
imagick
intl
json
libxml
maxminddb
mbstring
memcached
msgpack
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
Phar
posix
readline
redis
Reflection
session
shmop
SimpleXML
soap
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache
ddtrace
```

</details>

### OS info
```
NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
PRETTY_NAME="Ubuntu 20.04.3 LTS"
VERSION_ID="20.04"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

### Diagnostics and configuration
The trace-agent.log file looks like:

<details>
<summary>Click to expand</summary>

```2021-09-09 09:43:45 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:11494819935572506223 spanID:11494819935572506223 start:1631180623192299686 duration:19437 meta:<key:"env" value:"prod"
2021-09-09 09:43:45 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:11494819935572506223 spanID:11494819935572506223 start:1631180623192299686 duration:19437 meta:<key:"env" value:"prod" > 
2021-09-09 09:43:48 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:48 in normalize) | Fixing malformed trace. Service is empty (reason:service_empty), setting span.service=: traceID:9847748800527124456 spanID:9847748800527124456 start:1631180623513181802 duration:16520 meta:<key:"env" value:"prod" > 
2021-09-09 09:43:48 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:9847748800527124456 spanID:9847748800527124456 start:1631180623513181802 duration:16520 meta:<key:"env" value:"prod" > 
2021-09-09 09:43:48 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:9847748800527124456 spanID:9847748800527124456 start:1631180623513181802 duration:16520 meta:<key:"env" value:"prod" > 
2021-09-09 09:43:49 UTC | TRACE | DEBUG | (pkg/trace/stats/concentrator.go:180 in flushNow) | flushing bucket 1631180600000000000
2021-09-09 09:43:49 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:230 in flush) | Serializing 55 traces and 0 APM events.
2021-09-09 09:43:49 UTC | TRACE | DEBUG | (pkg/trace/stats/concentrator.go:190 in flushNow) | update oldestTs to 1631180610000000000
2021-09-09 09:43:49 UTC | TRACE | DEBUG | (pkg/trace/writer/stats.go:188 in func1) | Flushing 124 entries (buckets=1 client_payloads=1)
2021-09-09 09:43:49 UTC | TRACE | DEBUG | (pkg/trace/writer/stats.go:339 in recordEvent) | Flushed stats to the API; time: 21.98646ms, bytes: 13303
2021-09-09 09:43:49 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:299 in recordEvent) | Flushed traces to the API; time: 92.876578ms, bytes: 130305
2021-09-09 09:43:54 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:230 in flush) | Serializing 22 traces and 0 APM events.
2021-09-09 09:43:54 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:299 in recordEvent) | Flushed traces to the API; time: 47.798317ms, bytes: 149602
2021-09-09 09:43:59 UTC | TRACE | DEBUG | (pkg/trace/stats/concentrator.go:180 in flushNow) | flushing bucket 1631180610000000000
2021-09-09 09:43:59 UTC | TRACE | INFO | (pkg/trace/info/stats.go:108 in LogStats) | [lang:php lang_version:8.0.8 interpreter:fpm-fcgi tracer_version:0.63.0 endpoint_version:v0.4] -> traces received: 388, traces filtered: 0, traces amount: 8271184 bytes, events extracted: 0, events sampled: 0
2021-09-09 09:43:59 UTC | TRACE | WARN | (pkg/trace/info/stats.go:111 in LogStats) | [lang:php lang_version:8.0.8 interpreter:fpm-fcgi tracer_version:0.63.0 endpoint_version:v0.4] -> spans_malformed(resource_empty:6, service_empty:6, span_name_empty:6). Enable debug logging for more details.
2021-09-09 09:43:59 UTC | TRACE | INFO | (pkg/trace/info/stats.go:108 in LogStats) | [lang:php lang_version:8.0.8 interpreter:cli tracer_version:0.63.0 endpoint_version:v0.4] -> traces received: 6, traces filtered: 0, traces amount: 898 bytes, events extracted: 0, events sampled: 0
2021-09-09 09:43:59 UTC | TRACE | WARN | (pkg/trace/info/stats.go:111 in LogStats) | [lang:php lang_version:8.0.8 interpreter:cli tracer_version:0.63.0 endpoint_version:v0.4] -> spans_malformed(resource_empty:6, service_empty:6, span_name_empty:6). Enable debug logging for more details.
2021-09-09 09:43:59 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:230 in flush) | Serializing 24 traces and 0 APM events.
2021-09-09 09:43:59 UTC | TRACE | DEBUG | (pkg/trace/stats/concentrator.go:190 in flushNow) | update oldestTs to 1631180620000000000
2021-09-09 09:43:59 UTC | TRACE | DEBUG | (pkg/trace/writer/stats.go:188 in func1) | Flushing 137 entries (buckets=1 client_payloads=1)
2021-09-09 09:43:59 UTC | TRACE | DEBUG | (pkg/trace/writer/stats.go:339 in recordEvent) | Flushed stats to the API; time: 21.457163ms, bytes: 16459
2021-09-09 09:43:59 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:299 in recordEvent) | Flushed traces to the API; time: 43.963921ms, bytes: 53062
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:48 in normalize) | Fixing malformed trace. Service is empty (reason:service_empty), setting span.service=: traceID:722036051060467482 spanID:722036051060467482 start:1631180641140202013 duration:2012137013 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278330" > metrics:<key:"php.compilation.total_time_ms" value:923.218 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:722036051060467482 spanID:722036051060467482 start:1631180641140202013 duration:2012137013 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278330" > metrics:<key:"php.compilation.total_time_ms" value:923.218 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:722036051060467482 spanID:722036051060467482 start:1631180641140202013 duration:2012137013 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278330" > metrics:<key:"php.compilation.total_time_ms" value:923.218 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:48 in normalize) | Fixing malformed trace. Service is empty (reason:service_empty), setting span.service=: traceID:11741641463875513728 spanID:11741641463875513728 start:1631180641178814550 duration:1999030713 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278334" > metrics:<key:"php.compilation.total_time_ms" value:720.835 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:11741641463875513728 spanID:11741641463875513728 start:1631180641178814550 duration:1999030713 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278334" > metrics:<key:"php.compilation.total_time_ms" value:720.835 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:11741641463875513728 spanID:11741641463875513728 start:1631180641178814550 duration:1999030713 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278334" > metrics:<key:"php.compilation.total_time_ms" value:720.835 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:48 in normalize) | Fixing malformed trace. Service is empty (reason:service_empty), setting span.service=: traceID:3117779799319506901 spanID:3117779799319506901 start:1631180641149948670 duration:2246744394 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278333" > metrics:<key:"php.compilation.total_time_ms" value:988.979 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:3117779799319506901 spanID:3117779799319506901 start:1631180641149948670 duration:2246744394 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278333" > metrics:<key:"php.compilation.total_time_ms" value:988.979 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:3117779799319506901 spanID:3117779799319506901 start:1631180641149948670 duration:2246744394 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278333" > metrics:<key:"php.compilation.total_time_ms" value:988.979 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:48 in normalize) | Fixing malformed trace. Service is empty (reason:service_empty), setting span.service=: traceID:17559344363884548530 spanID:17559344363884548530 start:1631180641171278148 duration:2274211461 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278337" > metrics:<key:"php.compilation.total_time_ms" value:1023.219 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:17559344363884548530 spanID:17559344363884548530 start:1631180641171278148 duration:2274211461 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278337" > metrics:<key:"php.compilation.total_time_ms" value:1023.219 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:17559344363884548530 spanID:17559344363884548530 start:1631180641171278148 duration:2274211461 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278337" > metrics:<key:"php.compilation.total_time_ms" value:1023.219 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:48 in normalize) | Fixing malformed trace. Service is empty (reason:service_empty), setting span.service=: traceID:2570830624781944721 spanID:2570830624781944721 start:1631180641158642065 duration:2301651730 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278331" > metrics:<key:"php.compilation.total_time_ms" value:1050.122 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:73 in normalize) | Fixing malformed trace. Name is empty (reason:span_name_empty), setting span.name=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:2570830624781944721 spanID:2570830624781944721 start:1631180641158642065 duration:2301651730 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278331" > metrics:<key:"php.compilation.total_time_ms" value:1050.122 > 
2021-09-09 09:44:03 UTC | TRACE | DEBUG | (pkg/trace/agent/normalizer.go:84 in normalize) | Fixing malformed trace. Resource is empty (reason:resource_empty), setting span.resource=unnamed_operation: service:"unnamed-php-service" name:"unnamed_operation" traceID:2570830624781944721 spanID:2570830624781944721 start:1631180641158642065 duration:2301651730 meta:<key:"env" value:"prod" > meta:<key:"system.pid" value:"1278331" > metrics:<key:"php.compilation.total_time_ms" value:1050.122 > 
2021-09-09 09:44:04 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:230 in flush) | Serializing 42 traces and 0 APM events.
2021-09-09 09:44:04 UTC | TRACE | DEBUG | (pkg/trace/writer/trace.go:299 in recordEvent) | Flushed traces to the API; time: 85.588855ms, bytes: 117034
```
</details>

#### Output of phpinfo()
<details>
<summary>ddtrace PHP info</summary>

```
ddtrace

Datadog PHP tracer extension
For help, check out the documentation at https://docs.datadoghq.com/tracing/languages/php/
(c) Datadog 2020

Datadog tracing support => enabled
Version => 0.63.0
DATADOG TRACER CONFIGURATION => {"date":"2021-09-09T09:40:12Z","os_name":"Linux instance-vm 5.11.0-1017-gcp #19~20.04.1-Ubuntu SMP Thu Aug 12 05:25:25 UTC 2021 x86_64","os_version":"5.11.0-1017-gcp","version":"0.63.0","lang":"php","lang_version":"8.0.8","env":"prod","enabled":true,"service":"core","enabled_cli":false,"agent_url":"http://localhost:8126","debug":false,"analytics_enabled":false,"sample_rate":1.000000,"sampling_rules":null,"tags":{},"service_mapping":{},"distributed_tracing_enabled":true,"priority_sampling_enabled":true,"dd_version":null,"architecture":"x86_64","sapi":"cli","datadog.trace.request_init_hook":"/opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php","open_basedir_configured":false,"uri_fragment_regex":null,"uri_mapping_incoming":null,"uri_mapping_outgoing":null,"auto_flush_enabled":false,"generate_root_span":true,"http_client_split_by_domain":false,"measure_compile_time":true,"report_hostname_on_root_span":false,"traced_internal_functions":null,"auto_prepend_file_configured":false,"integrations_disabled":"default","enabled_from_env":true,"opcache.file_cache":null,"DDTRACE_REQUEST_INIT_HOOK":"'DDTRACE_REQUEST_INIT_HOOK=/opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php' is deprecated, use DD_TRACE_REQUEST_INIT_HOOK instead."}

                               Diagnostics                               
DDTRACE_REQUEST_INIT_HOOK => 'DDTRACE_REQUEST_INIT_HOOK=/opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php' is deprecated, use DD_TRACE_REQUEST_INIT_HOOK instead.
Diagnostic checks => failed

Directive => Local Value => Master Value
datadog.agent_host => localhost => localhost
datadog.autofinish_spans => Off => Off
datadog.cakephp_analytics_enabled => Off => Off
datadog.cakephp_analytics_sample_rate => 1 => 1
datadog.codeigniter_analytics_enabled => Off => Off
datadog.codeigniter_analytics_sample_rate => 1 => 1
datadog.curl_analytics_enabled => Off => Off
datadog.curl_analytics_sample_rate => 1 => 1
datadog.distributed_tracing => On => On
datadog.dogstatsd_port => 8125 => 8125
datadog.elasticsearch_analytics_enabled => Off => Off
datadog.elasticsearch_analytics_sample_rate => 1 => 1
datadog.eloquent_analytics_enabled => Off => Off
datadog.eloquent_analytics_sample_rate => 1 => 1
datadog.env => prod => prod
datadog.guzzle_analytics_enabled => Off => Off
datadog.guzzle_analytics_sample_rate => 1 => 1
datadog.integrations_disabled => default => default
datadog.laravel_analytics_enabled => Off => Off
datadog.laravel_analytics_sample_rate => 1 => 1
datadog.log_backtrace => On => On
datadog.lumen_analytics_enabled => Off => Off
datadog.lumen_analytics_sample_rate => 1 => 1
datadog.memcached_analytics_enabled => Off => Off
datadog.memcached_analytics_sample_rate => 1 => 1
datadog.mongo_analytics_enabled => Off => Off
datadog.mongo_analytics_sample_rate => 1 => 1
datadog.mysqli_analytics_enabled => Off => Off
datadog.mysqli_analytics_sample_rate => 1 => 1
datadog.nette_analytics_enabled => Off => Off
datadog.nette_analytics_sample_rate => 1 => 1
datadog.pdo_analytics_enabled => Off => Off
datadog.pdo_analytics_sample_rate => 1 => 1
datadog.phpredis_analytics_enabled => Off => Off
datadog.phpredis_analytics_sample_rate => 1 => 1
datadog.predis_analytics_enabled => Off => Off
datadog.predis_analytics_sample_rate => 1 => 1
datadog.priority_sampling => On => On
datadog.sampling_rate => 1 => 1
datadog.service => core => core
datadog.service_mapping => no value => no value
datadog.service_name => core => core
datadog.slim_analytics_enabled => Off => Off
datadog.slim_analytics_sample_rate => 1 => 1
datadog.symfony_analytics_enabled => Off => Off
datadog.symfony_analytics_sample_rate => 1 => 1
datadog.tags => no value => no value
datadog.trace.agent_attempt_retry_time_msec => 5000 => 5000
datadog.trace.agent_connect_timeout => 100 => 100
datadog.trace.agent_debug_verbose_curl => Off => Off
datadog.trace.agent_flush_after_n_requests => 10 => 10
datadog.trace.agent_flush_interval => 5000 => 5000
datadog.trace.agent_max_consecutive_failures => 3 => 3
datadog.trace.agent_port => 8126 => 8126
datadog.trace.agent_timeout => 500 => 500
datadog.trace.agent_url => no value => no value
datadog.trace.analytics_enabled => Off => Off
datadog.trace.auto_flush_enabled => Off => Off
datadog.trace.beta_high_memory_pressure_percent => 80 => 80
datadog.trace.bgs_connect_timeout => 2000 => 2000
datadog.trace.bgs_timeout => 5000 => 5000
datadog.trace.cakephp_analytics_enabled => Off => Off
datadog.trace.cakephp_analytics_sample_rate => 1 => 1
datadog.trace.cakephp_enabled => On => On
datadog.trace.cli_enabled => Off => Off
datadog.trace.codeigniter_analytics_enabled => Off => Off
datadog.trace.codeigniter_analytics_sample_rate => 1 => 1
datadog.trace.codeigniter_enabled => On => On
datadog.trace.curl_analytics_enabled => Off => Off
datadog.trace.curl_analytics_sample_rate => 1 => 1
datadog.trace.curl_enabled => On => On
datadog.trace.debug => Off => Off
datadog.trace.debug_curl_output => Off => Off
datadog.trace.debug_prng_seed => -1 => -1
datadog.trace.elasticsearch_analytics_enabled => Off => Off
datadog.trace.elasticsearch_analytics_sample_rate => 1 => 1
datadog.trace.elasticsearch_enabled => On => On
datadog.trace.eloquent_analytics_enabled => Off => Off
datadog.trace.eloquent_analytics_sample_rate => 1 => 1
datadog.trace.eloquent_enabled => On => On
datadog.trace.enabled => On => On
datadog.trace.generate_root_span => On => On
datadog.trace.global_tags => no value => no value
datadog.trace.guzzle_analytics_enabled => Off => Off
datadog.trace.guzzle_analytics_sample_rate => 1 => 1
datadog.trace.guzzle_enabled => On => On
datadog.trace.health_metrics_enabled => Off => Off
datadog.trace.health_metrics_heartbeat_sample_rate => 0.001 => 0.001
datadog.trace.http_client_split_by_domain => Off => Off
datadog.trace.laravel_analytics_enabled => Off => Off
datadog.trace.laravel_analytics_sample_rate => 1 => 1
datadog.trace.laravel_enabled => On => On
datadog.trace.lumen_analytics_enabled => Off => Off
datadog.trace.lumen_analytics_sample_rate => 1 => 1
datadog.trace.lumen_enabled => On => On
datadog.trace.measure_compile_time => On => On
datadog.trace.memcached_analytics_enabled => Off => Off
datadog.trace.memcached_analytics_sample_rate => 1 => 1
datadog.trace.memcached_enabled => On => On
datadog.trace.memory_limit => no value => no value
datadog.trace.mongo_analytics_enabled => Off => Off
datadog.trace.mongo_analytics_sample_rate => 1 => 1
datadog.trace.mongo_enabled => On => On
datadog.trace.mysqli_analytics_enabled => Off => Off
datadog.trace.mysqli_analytics_sample_rate => 1 => 1
datadog.trace.mysqli_enabled => On => On
datadog.trace.nette_analytics_enabled => Off => Off
datadog.trace.nette_analytics_sample_rate => 1 => 1
datadog.trace.nette_enabled => On => On
datadog.trace.pdo_analytics_enabled => Off => Off
datadog.trace.pdo_analytics_sample_rate => 1 => 1
datadog.trace.pdo_enabled => On => On
datadog.trace.phpredis_analytics_enabled => Off => Off
datadog.trace.phpredis_analytics_sample_rate => 1 => 1
datadog.trace.phpredis_enabled => On => On
datadog.trace.predis_analytics_enabled => Off => Off
datadog.trace.predis_analytics_sample_rate => 1 => 1
datadog.trace.predis_enabled => On => On
datadog.trace.report_hostname => Off => Off
datadog.trace.request_init_hook => /opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php => /opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php
datadog.trace.resource_uri_fragment_regex => no value => no value
datadog.trace.resource_uri_mapping => no value => no value
datadog.trace.resource_uri_mapping_incoming => no value => no value
datadog.trace.resource_uri_mapping_outgoing => no value => no value
datadog.trace.retain_thread_capabilities => Off => Off
datadog.trace.sample_rate => 1 => 1
datadog.trace.sampling_rules => no value => no value
datadog.trace.shutdown_timeout => 5000 => 5000
datadog.trace.slim_analytics_enabled => Off => Off
datadog.trace.slim_analytics_sample_rate => 1 => 1
datadog.trace.slim_enabled => On => On
datadog.trace.spans_limit => 1000 => 1000
datadog.trace.startup_logs => On => On
datadog.trace.symfony_analytics_enabled => Off => Off
datadog.trace.symfony_analytics_sample_rate => 1 => 1
datadog.trace.symfony_enabled => On => On
datadog.trace.traced_internal_functions => no value => no value
datadog.trace.url_as_resource_names_enabled => On => On
datadog.trace.warn_legacy_dd_trace => On => On
datadog.trace.web_analytics_enabled => Off => Off
datadog.trace.web_analytics_sample_rate => 1 => 1
datadog.trace.web_enabled => On => On
datadog.trace.wordpress_analytics_enabled => Off => Off
datadog.trace.wordpress_analytics_sample_rate => 1 => 1
datadog.trace.wordpress_enabled => On => On
datadog.trace.yii_analytics_enabled => Off => Off
datadog.trace.yii_analytics_sample_rate => 1 => 1
datadog.trace.yii_enabled => On => On
datadog.trace.zendframework_analytics_enabled => Off => Off
datadog.trace.zendframework_analytics_sample_rate => 1 => 1
datadog.trace.zendframework_enabled => On => On
datadog.version => no value => no value
datadog.web_analytics_enabled => Off => Off
datadog.web_analytics_sample_rate => 1 => 1
datadog.wordpress_analytics_enabled => Off => Off
datadog.wordpress_analytics_sample_rate => 1 => 1
datadog.yii_analytics_enabled => Off => Off
datadog.yii_analytics_sample_rate => 1 => 1
datadog.zendframework_analytics_enabled => Off => Off
datadog.zendframework_analytics_sample_rate => 1 => 1
ddtrace.cgroup_file => /proc/self/cgroup => /proc/self/cgroup
ddtrace.disable => Off => Off
ddtrace.request_init_hook => /opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php => /opt/datadog-php/dd-trace-sources/bridge/dd_wrap_autoloader.php
```

</details>

### Upgrading info

**0.54.0 --> 0.63.0**

### PR DESCRIPTION
**Pull request by bwoebi**
### Description

Fixes #1314.

### Readiness checklist
- [x] (only for Members) Changelog has been added to the release document.
- [x] Tests added for this feature/bug.

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.
- [ ] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: ext/php8/ddtrace.c [hunk 1]
```diff
@@ -424,6 +422,7 @@ static void dd_rinit_once(void) {
     ddtrace_coms_init_and_start_writer();
 }
 
+static pthread_once_t dd_rinit_config_once_control = PTHREAD_ONCE_INIT;
```
Comment (morrisonlevi) [inline ext/php8/ddtrace.c:425]: Why can't this share the once control for the other one-time rinit?

Comment (morrisonlevi) [reply ext/php8/ddtrace.c:425]: Can we leave a comment about this so I don't see it and try to refactor it?

File: ext/php8/ddtrace.c [hunk 2]
```diff
@@ -433,9 +432,15 @@ static PHP_RINIT_FUNCTION(ddtrace) {
         DDTRACE_G(disable) = 1;
     }
 
+    // ZAI config is always set up
```
Comment (bwoebi) [inline ext/php8/ddtrace.c:435]: The comment is already here @morrisonlevi

Comment (morrisonlevi) [reply ext/php8/ddtrace.c:435]: 👍🏻

**Review by morrisonlevi — APPROVED** [review]

**Pull request merged**